runtime/src/main/java/com/dylibso/chicory/runtime/TagInstance.java

```diff
@@ -1,22 +1,31 @@
 package com.dylibso.chicory.runtime;
 
+import com.dylibso.chicory.wasm.types.FunctionType;
 import com.dylibso.chicory.wasm.types.TagType;
 
 public class TagInstance {
 
     private final TagType tag;
-    private final Instance instance;
+    private FunctionType type;
 
-    public TagInstance(TagType tag, Instance instance) {
```

Comment:
This is an API breaking change, but I believe that the risk is really low.
`Tag`s are used by the EH proposal and ppl could not have been using this API for anything useful.
The EH PR might be impacted and iterate on this detail though.